h2o-algos/src/test/java/hex/grid/GridTest.java

```diff
@@ -89,6 +135,42 @@ public void testFaileH2OdParamsCleanup() throws FileNotFoundException {
     }
   }
 
+  @Test
+  public void testParallelGridSearch() {
```

Comment:
Please us this test to see the basic parallel approach that is now simply hardcoded for showcase.

The |hyperspace| = 12. There are quick-to-finish variants, as well as longer lasting ones (number of trees is the main factors affecting time in this test).